tests/testthat/test-sampler.R

```diff
@@ -46,19 +46,21 @@ test_that("sampler runs without error", {
 
   mod <- sircovid_model$odin_model(user = pars_model)
   compare <- sircovid_model$compare_model(mod, pars_obs, d)
+  
+  seeding_func <-sircovid_model$seeding_model(mod,d)
```

Comment:
Can we add a test with `pars_seeding` set

Reply:
Done